docs/source/posts/2021/2021-08-23-final-work-antriksh.rst

```diff
@@ -0,0 +1,197 @@
+
+-  **Resize Panel2D with WindowResizeEvent or from corner placeholder**
+
+   Currently, the size of the Panel2D is static and cannot be changed dynamically. The size is passed in during the initialization and cannot be changed easily at runtime. This PR adds support for resizing the Panel2D dynamically by adding a placeholder icon at the bottom right corner of the panel. This icon canbe click and dragged on to change the size accordingly. Other than this, the panel also retains a specific size ratio when the window is resized. This means if the window is resized in any direction the panel adapts itself w.r.t the updated size. This is done by adding relevant observers for the WindowResizeEvent and binding the relevant callback to it. Below is a quick demonstration:
+
+    .. image:: https://camo.githubusercontent.com/3b1bf6a1b6522a6079055ff196551362fcf89a41b35ac4b32315ce02333e496d/68747470733a2f2f692e696d6775722e636f6d2f3837504e3754512e676966
+        :width: 200
+        :height: 200
+
+   *Pull Requests:*
+
+   - **Resize Panel2D with WindowResizeEvent or from corner placeholder:**  https://github.com/fury-gl/fury/pull/446
+
+-  **Added the watcher class to UI**
+
+   This PR adds support for a watcher class in the UI elements. The purpose of this class is to monitor a particular attribute from the UI element after it has been added to the scene. If the attribute changes in the real time, a user defined callback is triggered and the scene is force rendered.
+
+   *Pull Requests:*
+
+   - **Added wathcer class to the UI sub-module:**  https://github.com/fury-gl/fury/pull/448
+
+-  **Added support for borders in Panel2D**
+
+   The Panel2D previously, didn't support any sort of effect, the main reason behind this is that, all UI elements are individual entities that are comprised of different actors. These are not the widgets provided by vtk and in order to have some effects provided by vtk shaders must be involved. This obviously makes the whole system very complicated. The border on the other hand uses 4 Rectangle2Ds to draw the 4 borders. This makes the whole process easier but makes the Panel2D very performance heavy as we are adding 5 actors to the scene. Future iterations will replace these rectangles by textures, that way we don't compromise performance and we can have different patterns in the border. Below is a demonstration:
+
+   .. image:: https://user-images.githubusercontent.com/54466356/121709989-bd340280-caf6-11eb-9b8a-81c65260d277.png
+        :width: 200
+        :height: 200
+ 
+   *Pull Requests:*
+
+   - **Added support for borders in Panel2D:**  https://github.com/fury-gl/fury/pull/441
+
+-  **GSoC weekly Blogs**
+
+    Weekly blogs were added for FURY's Website.
+
+    *Pull Requests:*
+
+    - **First Evaluation:** https://github.com/fury-gl/fury/pull/477
+
+    - **Second Evaluation:** https://github.com/fury-gl/fury/pull/494
+
+Timeline
+--------
+
```

Comment:
The table did not work, can you check why ? See below:
<img width="970" alt="Capture d’écran 2021-09-11 à 15 24 47" src="https://user-images.githubusercontent.com/23106443/132949361-18fec59d-e0c4-43a5-8765-ad7a9b30fc39.png">

Reply:
The table had some formatting issues. It should be fine now.